codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/CustomizationConfig.java

```diff
@@ -201,6 +201,11 @@ public class CustomizationConfig {
      */
     private boolean delegateAsyncClientClass;
 
+    /**
+     * Whether to skip generating endpoint tests from endpoint-tests.json
+     */
+    private boolean skipEndpointTestGeneration;
```

Comment:
For packages like `protocol-tests` and `codegen-generated-classes-test` where we don't need to test them.

Reply:
Are we not testing is because its redundant or if its is taking more time to execute the test cases

Reply:
It doesn't make sense to test rules for those package because they're just placeholders.